build/README.md

```diff
@@ -33,3 +33,16 @@ However, it will fail on Windows due to the [`pango`](https://anaconda.org/conda
 
 Because the build process is dependent on having the appropriate version of the `manubot` Python package, it is necessary to use the version specified in `environment.yml`.
 The latest `manubot` release on PyPI may not be compatible with the latest version of this rootstock repository.
+
+## Building PDFs
+
+If Docker is available, `build.sh` uses the [Athena](https://www.athenapdf.com/) [Docker image](https://hub.docker.com/r/arachnysdocker/athenapdf) to build the PDF.
+Otherwise, `build.sh` uses [WeasyPrint](https://weasyprint.org/) to build the PDF.
+It is common for WeasyPrint to generate many warnings and errors that can be safely ignored.
+Examples are shown below:
+```
+WARNING: Ignored `pointer-events: none` at 3:16, unknown property.
+WARNING: Ignored `font-display:auto` at 1:53114, descriptor not supported.
+ERROR: Failed to load font at "https://use.fontawesome.com/releases/v5.7.2/webfonts/fa-brands-400.eot#iefix"
+WARNING: Expected a media type, got only/**/screen
+```
```

Comment:
Perhaps we should say something like:

> Setting `BUILD_PDF=false` will disable all PDF creation.

Reply:
At the top of the file we have
> However, setting the `BUILD_PDF` environment variable to `false` will suppress PDF output.
For example, run local builds using the command `BUILD_PDF=false bash build/build.sh`.

Should we move that or leave it there?

Reply:
Ah I missed that from the diff. Your call.

Reply:
I'll leave it for now. Feel free to move it later if we make another pass through the documentation.